### ISSUE TITLE
Support for 16.10?

### ISSUE THREAD

**Issue by skorokithakis** [tmate-io/tmate-ppa#5]
2017 is almost upon us, can we get support for 16.10 in the PPA?

**Comment by nviennot** [comment]
@chilicuil I'm not sure if you have some bandwidth these days, if you do, can you take a look for 16.10?
Thank you :)

**Comment by chilicuil** [comment]
Indeed, @nviennot , @skorokithakis =) , I've just updated the ppa to allow binaries for 16.10, it compiled fine but I've not tested them on a real 16.10 system, could someone test them (tmux/tmux-nightly) and tell if they work?
